### PR TITLE
Update body_bec_mobile_solicitation.yml

### PR DESCRIPTION
**Pull request by aidenmitchell**

### CODE REVIEW

File: detection-rules/body_bec_mobile_solicitation.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ source: |
                      "(email|send|shoot|give|provide|attach|confirm|) (me|your)"
   )
   and regex.contains(body.current_thread.text,
-                     "(cell(/s?phone)?|mobile.{0,10}(phone|number|#|no)|whatsapp)"
+                     "(cell(/s?phone)?|mobile.{0,10}|contact.{0,10}(phone|number|#|no)|whatsapp)"
```
Comment (jkamdjou) [inline detection-rules/body_bec_mobile_solicitation.yml:13]:
```suggestion
                     "(cell(/s?phone)?|(mobile.{0,10}|contact.{0,10})(phone|number|#|no)|whatsapp)"
```
does this need to be this to maintain current parity? assuming this is for FNs

Comment (aidenmitchell) [reply detection-rules/body_bec_mobile_solicitation.yml:13]: Don't think so, it's supposed to be something like `(cellphone OR mobile... OR contact...)` AND `(phone|number|#|no)|whatsapp`

Comment (morriscode) [reply detection-rules/body_bec_mobile_solicitation.yml]: Both regex's were looking a bit wonky. Updated both

**Comment by morriscode** [comment]
FWIW this rule was super busted due to a floating pipe. Going to take a minute to fix it.

**Review by morriscode — APPROVED** [review]

**Pull request merged**